src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -279,34 +277,8 @@ private static FilePath tempDir(FilePath ws) {
             this.toolName = toolName;
         }
 
-        private void stopContainer(StepContext context) {
-            Launcher launcher;
-            TaskListener listener;
-            try {
-                launcher = context.get(Launcher.class);
-                listener = context.get(TaskListener.class);
-            } catch (Exception x2) {
-                LOGGER.log(Level.WARNING, null, x2);
-                return;
-            }
-            try {
-                EnvVars launcherEnv = context.get(EnvVars.class);
-                Node node = context.get(Node.class);
-                destroy(container, launcher, node, launcherEnv, toolName);
-            } catch (Exception x) {
-                x.printStackTrace(listener.error("Could not kill container " + container));
```

Comment:
As of https://github.com/jenkinsci/workflow-step-api-plugin/pull/5 this should go into the “suppressed” chain.